CmakeLists.txt

```diff
@@ -40,13 +40,19 @@ set(RGM_SOURCES
     Models/TreeModel.cpp
     Models/SpriteModel.cpp
     Models/ProtoModel.cpp
+    Models/ModelMapper.cpp
```

Comment:
I wouldn't know that reason and this is actually getting painful for me to update both lists of sources in the qmake pro and the cmake lists, but I would like to continue to support building with both. Mainly because I use qtcreator when writing pull requests even though we are building releases with MSVC.

Reply:
You can just `glob()` the directories you want. I think there's even an exclude mechanism so you can colocate test sources with the same extension.

Reply:
Fundies says don't glob, because cmake ppl annoying about it and will cry. C++ sucks at deps, we can come back to this later.